builtins/amp-img.js

```diff
@@ -38,6 +38,20 @@ export class AmpImg extends BaseElement {
 
     /** @private {?../src/srcset.Srcset} */
     this.srcset_ = null;
+
+    /** @private {!Array<string>} */
+    this.attributesToPropagate_ = ['alt', 'referrerpolicy', 'aria-label',
+      'aria-describedby', 'aria-labelledby'];
+  }
+
+  /** @override */
+  attributeChangedCallback(name, unusedOldValue, unusedNewValue) {
+    if (name === 'src') {
+      this.srcset_ = srcsetFromElement(this.element);
+      this.updateImageSrc_();
+    } else if (this.img_ && this.attributesToPropagate_.indexOf(name) >= 0) {
+      this.propagateAttributes(name, this.img_);
```

Comment:
At this point, `Bind` will have already called `element.setAttribute(name, newValue)` so this should still work for adding new attributes. 

Good point about removing attributes though, added a new optional param to support that.